documentation/release_notes.markdown

```diff
@@ -10,6 +10,16 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.12.1
+
+Released 14 July 2020
+
+### Enhancements
+
+- Users who’ve started logging jolokia debug info will now have to enable a setting in the config file to get that back. New users that want to get debug info will have to add that setting, in additional updating logback.xml as they would have had to do before. Now, by default, attempting to access uncreated metrics will quietly continue rather than throw exceptions. The throwing exceptions behavior will return if enabling debug output. [TK-488](https://tickets.puppetlabs.com/browse/TK-488)
+
+- The metrics routes are now covered by trapperkeeper-authorization, so access can be controlled from auth.conf (or the authorization section in general in the tk configuration). There is still a default embedded jolokia-access.xml which restricts access to localhost by ip which needs to be overridden by software intending to use tk-auth instead. [TK-489](https://tickets.puppetlabs.com/browse/TK-489)
```

Comment:
people - users - intend to do thing, software not so much.

Reply:
what

Reply:
"...by IP which needs to updated, in order to use tk-auth's rules instead."

Updated to allow anything?